src/Microsoft.ML.Data/Data/Conversion.cs

```diff
@@ -252,9 +252,6 @@ private Conversions()
             AddIsNA<R8>(IsNA);
             AddIsNA<BL>(IsNA);
             AddIsNA<TX>(IsNA);
-            AddIsNA<TS>(IsNA);
-            AddIsNA<DT>(IsNA);
-            AddIsNA<DZ>(IsNA);
```

Comment:
As far as I see you still need these. They will have trivial implementation, but you still need them.

So previously we had all types be *capable* of having missing values. Now you're changing that. So one of two things has to happen:

1. There must be some indicating somewhere that the `ColumnType` does not have missing types or
2. We must continue to support these things here with trivial implementations that just return `false` everywhere.